### PR TITLE
TLPORTAL-448: fix issue not finding multiple background image files

### PR DESCRIPTION
**Pull request by dlhaines**
This is a minimal fix for just this issue since it is the only instance found.  Other situations where the problem might occur will be addressed in JIRA-451 for Dash 2.2.

### CODE REVIEW

File: server/spec/test_external_resources_file.rb [hunk 1]
```diff
@@ -38,16 +38,31 @@ def test_constructor
     assert_equal @erf.resources_base_directory, @resources_dir, "set external resources directory name correctly"
   end
 
-  def test_list_resources_top
+  def test_list_resources_top_nil
```
Comment (zqian) [inline server/spec/test_external_resources_file.rb]: did not see the new function name get used in the diff.

File: server/spec/test_external_resources_file.rb [hunk 2]
```diff
@@ -38,16 +38,31 @@ def test_constructor
...
+    parsed_file_list = JSON.parse(@erf.get_resource "")
+    assert_equal 2,parsed_file_list.length,"list resources in top level directory"
+  end
+
   def test_list_resources_images
     parsed_file_list = JSON.parse(@erf.get_resource "image")
     assert_equal 8,parsed_file_list.length,"list resources in images directory"
   end
 
+  def test_list_resources_images_nil_file_name
```
Comment (zqian) [inline server/spec/test_external_resources_file.rb]: Does this mean that the count of images with either nil or empty file name are both of number 8?

Comment (dlhaines) [inline server/spec/test_external_resources_file.rb]: It means that if no specific file is requested a list of files will be returned.  I'm conflicted about how much documentation to put in test files.

Comment (gsilver) [inline server/spec/test_external_resources_file.rb]: LGTM - thanks for talking me through it.

**Pull request merged**